media/css/mozorg/home/home-variant.scss

```diff
@@ -0,0 +1,104 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import 'home.scss';
+
+#icon-sprite {
+    display: none;
+}
+
+#who-innovate-wrapper {
+    h2 {
+        @include font-size(32px);
+    }
+
+    .icon-links a {
+        @include trailing-arrow;
+        font-weight: bold;
+        text-decoration: none;
+
+        &:after {
+            @include transition(right 150ms ease);
+            position: relative;
+            right: 0;
+        }
+
+        &:hover:after,
+        &:focus:after {
+            right: -4px;
+        }
+    }
+
+    .icon {
+        display: none;
+    }
+
+    #who a:link,
+    #who a:visited {
+        color: #d93a33;
+    }
+
+    #who .icon {
+        background-color: #d93a33;
+
+        use {
+            fill: #fff;
+        }
+    }
+
+    #innovate .icon {
+        background-color: #fff;
+
+        use {
+            fill: #d93a33;
+        }
+    }
+}
+
+@media #{$mq-tablet} {
+    .js .variant-b {
+        .icon-links li {
+            opacity: .25;
+        }
+    }
+
+    #who-innovate-wrapper {
+        .icon-links li {
+            margin: 30px 0;
+            position: relative;
+        }
+
+        .icon-links a {
+            &:before {
+                display: none;
+            }
+
+            &:hover .icon,
+            &:focus .icon {
+                padding: 7px;
+                left: -27px;
+                top: -14px;
+            }
+        }
+
+        .icon {
+            display: inline-block;
+            width: 45px;
+            height: 45px;
+            padding: 5px;
+            background-color: silver;
+            border-radius: 50%;
+            position: absolute;
+            left: -25px;
+            top: -12px;
+            transition: left 150ms ease, top 150ms ease, padding 150ms ease;
```

Comment:
Good idea. I think repainting the padding while also shifting position makes for a tiny bit of jank, but I didn't want to scale the whole icon (and also wanted to avoid wrapping it a span). But I think I can use a pseudo-element for the solid circle and the image can just be on top of it. I think it'll work and should hopefully be smoother, and a transform-origin means I won't have to mess with repositioning.